source/Item.hx

```diff
@@ -0,0 +1,86 @@
+package;
+import flixel.FlxG;
+import flixel.FlxObject;
+import flixel.FlxSprite;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import haxe.Timer;
+
+
+/**
+ * ...
+ * @author Cameron Yuan
+ */
+class Item extends FlxSprite
+{
+	private var player_trace:Player;
```

Comment:
Keeping an internal copy of `Player` feels really awkward.  If it is just to access things like player facing, could you maybe do that through `FlxG` instead?